pkg/registry/core/rest/storage_core.go

```diff
@@ -268,9 +268,11 @@ func (s componentStatusStorage) serversToValidate() map[string]componentstatus.S
 			port = 2379
 		}
 		// TODO: etcd health checking should be abstracted in the storage tier
```

Comment:
It's unfortunate we can't fix this here, instead of requiring a new set of flags.